server/features/featureConstants.js

```diff
@@ -0,0 +1,5 @@
+const EDIT = "edit";
+const DELETE = "delete";
+const CREATE = "create";
+
+module.exports = {EDIT, CREATE, DELETE};
```

Comment:
Not sure about filename, what about `fetureOperations`?  `Constants` may become a big map which contains basically everything